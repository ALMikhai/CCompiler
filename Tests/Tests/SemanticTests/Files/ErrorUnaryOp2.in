int f() {
    int b = 1;
    int a = *b;
}
