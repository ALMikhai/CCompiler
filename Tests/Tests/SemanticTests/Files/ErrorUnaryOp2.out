(3:14): semantic error: invalid type argument of unary
